test/javaTests/pom.xml

```diff
@@ -38,7 +38,7 @@
 		<dependency>
 			<groupId>org.testng</groupId>
 			<artifactId>testng</artifactId>
-			<version>6.3.1</version>
+			<version>7.6.1</version>
```

Comment:
It looks like version [7.6.1](https://mvnrepository.com/artifact/org.testng/testng/7.6.1) also has some vulnerabilities. Should we update to version [7.7.0](https://mvnrepository.com/artifact/org.testng/testng/7.7.0) instead?